### PR TITLE
feat(label): added label component to library

### LINKED ISSUE #65: Labels
## 🚀 Feature Proposal

Add Labels to the component library.

## Motivation

Used to display text.

## Example

Labeling a form field.

## Related
#32

### PR DESCRIPTION
**Pull request by gqycloud**
Fixes #65 .

* Added label to component library
* Also auto updated doc-gen md files

### CODE REVIEW

File: src/components/Label/Label.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import FormLabel from 'react-bootstrap/FormLabel'
+
+interface Props {
```
Comment (jackcmeyer) [inline src/components/Label/Label.tsx]: I think we might want to support `htmlFor` so we can tie form controls to their labels if necessary.

What do do you think?

File: src/components/Label/Label.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  /** Text to display in label */
+  text: string
+}
+
+/**
+ * Labels are used to display textnpm
+ */
+class Label extends Component<Props, {}> {
```
Comment (jackcmeyer) [inline src/components/Label/Label.tsx]: Would this component be better as a functional component?

File: test/badge.test.tsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ describe('Badge', () => {
     expect(bootstrapBadge.props().variant).toEqual('secondary')
   })
 
-  it('Badge can redner children elements', () => {
+  it('Badge can render children elements', () => {
```
Comment (jackcmeyer) [inline test/badge.test.tsx]: Thanks for fixing the typo :)

**Comment by jackcmeyer** [comment]
@gqycloud Thanks for your work! Just a couple points of feedback!

File: test/badge.test.tsx (hunk 3 above)
Comment (gqycloud) [reply test/badge.test.tsx]: no problems (:

File: src/components/Label/Label.tsx (hunk 2 above)
Comment (gqycloud) [reply src/components/Label/Label.tsx]: Initially I was going to use a functional component here but after looking at some of the other bootstrap components (i.e textInput, textField, radio, badge, ect) I noticed that we tend towards using React Components. I wasn't sure if it was an explicit decision or not so I choose to follow the paradigm in the other components on this initial cut. 

I'm happy to change this component to a functional one for now though, what are you thoughts on adding a ticket for refactoring the other bootstrap components to functional components?

File: src/components/Label/Label.tsx (hunk 1 above)
Comment (gqycloud) [reply src/components/Label/Label.tsx]: Happy to add this prop, on that note what are you thoughts on adding a column prop to support horizontal forms? Do we have any plans to use horizontal forms or any plans to in the future?

File: src/components/Label/Label.tsx (hunk 2 above)
Comment (jackcmeyer) [reply src/components/Label/Label.tsx]: I actually refractored  most of the components last night 😀 https://github.com/HospitalRun/components/pull/85.

File: src/components/Label/Label.tsx (hunk 1 above)
Comment (jackcmeyer) [reply src/components/Label/Label.tsx]: No plans that I no of yet. But if it’s simple enough to add, no harm in it!

**Pull request merged**